Sources/SimpleCurrencyConverter/ExchangeRatesNetworkManager.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+
+public enum ExchangeRatesError: Error {
+    case failedToGetResponse
+    case failedToConfigureURL
+    case invalidResponse
+    case failedToDecodeResponse
+}
+
+final class ExchangeRatesNetworkManager {
+    
+    private let session: URLSession
+    
+    init(session: URLSession) {
+        self.session = session
+    }
+    
+    @available(iOS 13.0.0, *)
```

Comment:
Maybe it's better to set the deployment target to iOS 13 in Package.swift.
I do not remember if OJT requires iOS 12 support.

Reply:
There are no requirements for iOS version support. So I will set up a package for iOS 13. Thanks.